src/polychord/mpi_utils.F90

```diff
@@ -693,6 +693,70 @@ function more_points_needed(mpi_information)
 
     end function more_points_needed
 
+    !> Request specific live points
+    ! administrator                     ----> worker
+    ! request_this_point(live_point)   point_needed -> true
+    ! no_more_points (defined above)   point_needed -> false
+    ! 
+
+    !> Request this point
+    !!
+    !! This subroutine is used by the root node to request a specific live point
+    subroutine request_this_point(live_point,mpi_information,worker_id)
```

Comment:
Not fully sold on `request_this_point` as a name, ideas welcome

Reply:
Let's change to request_live_point